core-api/src/main/java/com/optimizely/ab/optimizelyconfig/OptimizelyFeature.java

```diff
@@ -32,6 +32,11 @@
     private List<OptimizelyExperiment> deliveryRules;
     private List<OptimizelyExperiment> experimentRules;
 
+    /**
+     * @Deprecated The experimentsMap has been deprecated. </br>
+     *              Use experimentRules and deliveryRules instead.
+     * */
+    @Deprecated()
     private Map<String, OptimizelyExperiment> experimentsMap;
```

Comment:
This does not show deprecation note when I open the doc. Can you check? Also it use different deprecation notation. You can make it consistent (for compatibility issue for android support?).

Reply:
Sure, let me take a look.

Reply:
@jaeopt Updated to follow suite with Android.